src/gemini/index.ts

```diff
@@ -0,0 +1,80 @@
+import { FirebaseError } from "../error";
+import { logger } from "../logger";
+import * as fs from "fs";
+import * as path from "path";
+import { promptOnce } from "../prompt";
+import { execSync } from "child_process";
+import * as clc from "colorette";
+
+const { GoogleGenerativeAI } = require("@google/generative-ai");
+
+// TODO(christhompson): Do we have an endpoint that's open and doesn't require a project w/ billing?
+// Estimated QPS is around 1.
+// TODO(christhompson): Add project ID for this.
+// TODO(christhompson): Add preamble information about command flags.
+// TODO(christhompson): Figure out how to test this.
+const generationConfig = {
+  maxOutputTokens: 200,
+  temperature: 0.9,
+  topP: 0.1,
+  topK: 16,
+};
+
+const genAI = new GoogleGenerativeAI("AIzaSyDDqii7EVJbMgfC3vhp3ER2o-EYa9qOSQw");
```

Comment:
Do you guys use a linter/formatter on the CLI?  Generally recommend prettier, but the line length and formatting here seems a bit long.